app/src/main/AndroidManifest.xml

```diff
@@ -57,6 +57,24 @@
             android:label="@string/app_name"
             android:screenOrientation="portrait"/>
 
+        <activity
+            android:name=".reminders.ReminderActivity"
+            android:label="@string/app_name"
+            android:screenOrientation="portrait"/>
+
+        <!-- These control local reminder notification scheduling and display -->
+        <receiver android:name="org.sagebionetworks.research.sageresearch.reminders.MpReminderAlarmReceiver" />
+        <receiver android:name=".reminders.MpReminderRescheduleReceiver">
+            <intent-filter>
+                <action android:name="android.intent.action.QUICKBOOT_POWERON" />
```

Comment:
Can you add comments on why these are needed?
I understand BOOT_COMPLETE and TIMEZONE_CHANGED but am not sure on the rest.

Reply:
Good point.  They're kind of a mystery as they are now.

Reply:
I added some documentation